doc/source/conf.py

```diff
@@ -101,20 +101,20 @@
         reldir = os.path.relpath(dirname, source_path)
         for fname in fnames:
             if os.path.splitext(fname)[-1] in (".rst", ".ipynb"):
-                fname = os.path.relpath(os.path.join(dirname, fname), source_path)
+                fname_new = os.path.relpath(os.path.join(dirname, fname), source_path)
```

Comment:
thanks for working on this - naming variables is hard, I'll take a closer look this week

Reply:
No problem, I will apply the rule of prefixing with an "_". My bad I should've asked earlier if there was a better standard way.
Yesterday I had some spare time and completed the gradual enabling of "pandas/plotting/*" and "pandas/tests/*". Currently halfway for completing "pandas/io/*". Is it fine if I put everything in a single pull request? I think I can manage to make all the changes.

Reply:
separate would be easier to review, would prefer that if possible :)

Reply:
>  I will apply the rule of prefixing with an "_".

actually, please wait, sorry - we should probably favour descriptive names

Reply:
maybe this one could be `rel_fname`?

Reply:
I understand the inner battle between descriptive names and verbosity :D.
I will will rethink a bit more about the name and update this PR ok? 
Then for the other sections of "tp be gradually enabled" I can generate other PRs from different branches for it.